packages/govern-server/src/core/utils/network.ts

```diff
@@ -0,0 +1,84 @@
+import { ErrorInvalidNetwork } from '../errors'
```

Comment:
Not sure how the structure is but we will use I think anyways all of those functions here. This means it will get bundled anyways and we could forward the creation of nice TS classes. 

What do you think about creating out of it a value object class?

``` typescript
const network = new Network('mainnet');
console.log(network.chainId);

const network = new Network(netObj);
console.log(network.chainId);

console.log(network.toObject());
> { ... }
```